go-controller/pkg/ovn/master.go

```diff
@@ -634,6 +634,12 @@ func (oc *Controller) syncGatewayLogicalNetwork(node *kapi.Node, l3GatewayConfig
 		return fmt.Errorf("failed to allocate join switch port IP address for node %s: %v", node.Name, err)
 	}
 
+	if l3GatewayConfig.EgressGWInterfaceID != "" {
+		oc.setExGatewayBridgeForNode(node.Name, true)
```

Comment:
what about node delete?

Reply:
I forgot about it :-)
In any case, it would have been some garbage in the map and nothing else. If moving to the lister implementation this will be fixed